CHANGELOG.md

```diff
@@ -1,3 +1,284 @@
+
+### Supported Kubernetes Versions
+- 1.20.13
+- 1.20.14
+- 1.21.8
+- 1.22.5
+
+
+### New and Enhanced Features
+
+#### External Cluster
+- endpoint to get external cluster upgrades `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/upgrades` ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
+- new endpoint to list external cluster machine deployments `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments` ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
+- Add PATCH endpoint for external cluster `PATCH /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}`([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
+- new endpoint to list External MD Metrics: `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments/{machinedeployment_id}/nodes/metrics` ([#8239](https://github.com/kubermatic/kubermatic/issues/8239))
+- Manage AKS Clusters natively. Extend external cluster functionality to import existing AKS cluster ([#8244](https://github.com/kubermatic/kubermatic/issues/8244))
+- Create AKS nodePool ([#8399](https://github.com/kubermatic/kubermatic/issues/8399))
+- Upgrade AKS NodePool Kubernetes version  ([#8349](https://github.com/kubermatic/kubermatic/issues/8349))
+- Scale AKS NodePool ([#8349](https://github.com/kubermatic/kubermatic/issues/8349))
+- Get AKS NodePool machineDeployments and nodes([#8349](https://github.com/kubermatic/kubermatic/issues/8349))
+- Delete AKS NodePool ([#8349](https://github.com/kubermatic/kubermatic/issues/8349))
+- Modified GET `/providers/aks/clusters` => `/projects/{project_id}/providers/aks/clusters` ([#8423](https://github.com/kubermatic/kubermatic/issues/8423))
+- GET `/providers/eks/clusters` => `/projects/{project_id}/providers/eks/clusters` ([#8423](https://github.com/kubermatic/kubermatic/issues/8423))
+- Adds GET Endpoint `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/kubeconfig` ([#8122](https://github.com/kubermatic/kubermatic/issues/8122))
+
+
+#### Operating System Management (OSM)
+- Integrating Operating System Manager in KKP ([#8473](https://github.com/kubermatic/kubermatic/issues/8473))
+- Validating webhooks for OSM resources, OperatingSystemProfile and OperatingSystemConfig ([#8503](https://github.com/kubermatic/kubermatic/issues/8503))
+- Deploy OSM CRDs in KKP seed clusters ([#8528](https://github.com/kubermatic/kubermatic/issues/8528))
+- Introduce new field enableOperatingSystemManager in Cluster and ClusterTemplate CRD specification:  
+  - use-osm flag is passed to machine-controller and machine-controller webhooks in case OSM is enabled  
+  - API integration for OSM ([#8523](https://github.com/kubermatic/kubermatic/issues/8523))
+- CRDs for OSP and OSC are updated in kubermatic-operator  
```

Comment:
GH won't let me do suggestions, but this should be one sentence

Reply:
done :)